data/src/test/java/io/gnosis/data/repositories/TokenRepositoryTest.kt

```diff
@@ -125,51 +123,6 @@ class TokenRepositoryTest {
         assertEquals("4.021".toBigDecimal(), balances.items[2].fiatBalance)
     }
 
-    @Test
-    fun `loadCollectiblesOf (address) should return grouped list of collectibles`() = runBlocking {
```

Comment:
Should we have a new test instead of this?

Reply:
no, because we are not grouping collectibles anymore and rely on backend to provide them in correct order